KarhooSDK.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
why are we deleting the `Package.resolved` now?

Reply:
CICD had problem with resolving locked packages